src/main/java/roomescape/TimeController.java

```diff
@@ -0,0 +1,50 @@
+package roomescape;
+
+import org.slf4j.ILoggerFactory;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@Controller
+public class TimeController {
+    private TimeQueryingDAO queryingDAO;
+    public TimeController(TimeQueryingDAO queryingDAO) {
+        this.queryingDAO = queryingDAO;
+    }
+
+    @GetMapping("/times")
+    public ResponseEntity<List<Time>> read() {
+        return ResponseEntity.ok(queryingDAO.getTimes());
+    }
+
+    @PostMapping("/times")
+    public ResponseEntity<ResponseDto> create (@RequestBody TimeRequestDto request) {
+
+        String time = request.getTime();
+        long id = queryingDAO.createTime(time);
+
+        URI location = URI.create("/times/" + id);
+        Time newTime = new Time(id, time);
+
+        ResponseDto response = new ResponseDto(HttpStatus.CREATED.value(), "시간이 성공적으로 추가되었습니다.", newTime);
+        return ResponseEntity.created(location).body(response);
+    }
+
+    @DeleteMapping("/times/{id}")
+    public ResponseEntity<Void> delete (@PathVariable Long id) {
+        int rowsAffected = queryingDAO.deleteTimeById(id);
+
+        if(rowsAffected > 0){
+            return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+        }else {
+            throw new NotFoundReservationException("Time with id " + id + " not found." );
+        }
+    }
```

Comment:
```suggestion
if (rowsAffected == 0) {
    throw new NotFoundReservationException();
}

return ResponseEntity.noContent().build();
```

그냥 생각나서 적어보는 것인데요 저는 보통 else 를 쓰지 않는 것이 가독성이 좋다고 느껴지더라구요.
+) 그리고 미션 제출 전 포맷팅 한 번씩 해주세요~